financial-templates-lib/ExpiringMultiPartyEventClient.js

```diff
@@ -1,20 +1,19 @@
 const { delay } = require("./delay");
-const { Logger } = require("./logger/Logger");
-const { LiquidationStatesEnum } = require("../common/Enums");
 
 // A thick client for getting information about an ExpiringMultiParty events.
 // This client is kept separate from the main ExpiringMultiPartyClient to keep
 // a clear separation of concerns and to limit the overhead from querying chain necessarily.
 // If no updateThreshold is specified then default to updating every 60 seconds.
 class ExpiringMultiPartyEventClient {
-  constructor(abi, web3, empAddress, updateThreshold = 60) {
+  constructor(logger, abi, web3, empAddress, updateThreshold = 60) {
```

Comment:
This PR changes how the logger is initialized in all off-chain monitoring infrastructure: rather than each bot having it's own logger instance it is rather injected. This dependency injection facilitates logger output testing by passing a custom logger which listens for logging events.

Reply:
I like this as it is more modular